src/components/Book/Book.js

```diff
@@ -0,0 +1,16 @@
+import './Book.css';
+
+const Book = () => (
+  <div className="book">
+    <p>Action</p>
+    <h3>The Hunger Games</h3>
+    <p>Suzanne Collins</p>
```

Comment:
- [ ] Nice job for displaying the title and author of the book however requirements say the `Book` should take the book data as a prop and renders the title and the author of the book. Please let's considered this requirement too.